app/dashboard/models.py

```diff
@@ -651,7 +663,7 @@ def psave_bounty(sender, instance, **kwargs):
     }
 
     instance.idx_status = instance.status
-    instance._val_usd_db = instance.value_in_usdt if instance.value_in_usdt else 0
+    instance._val_usd_db = instance.value_in_usdt_now if instance.value_in_usdt_now else 0
```

Comment:
> _val_usd_db

i wonder if we should factor this into two fields, `_val_usd_db` and `_val_usd_db_now` ?

Reply:
done.